scripts/site-build.sh

```diff
@@ -2,6 +2,30 @@
 set -e
 HUGO_DOCKER_IMAGE="klakegg/hugo:ext-alpine"
 ASSET_DIR=${1:-src/query/generated/assets/openapi}
+
+# First generate old versions
+docsVersions=$(git tag -l 'docs/*')
+for docVersion in $docsVersions
+do
+        echo "Building $docVersion"
+        git checkout -q "tags/$docVersion"
+        # Copy over OpenAPI doc.
```

Comment:
nit: Since the code for generating the docs includes a lot of preparation steps that are repeated both for generating tagged doc versions and the latest one, would be nice to extract it into functions.